Facebook.iOS/source/AccountKit/AccountKit.csproj

```diff
@@ -68,309 +72,18 @@
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\tr.lproj\AccountKitCountryCode.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\tr.lproj\AccountKitCountryCode.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\ur.lproj\AccountKit.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\ur.lproj\AccountKit.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\ur_PK.lproj\AccountKit.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\ur_PK.lproj\AccountKit.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\ur_PK.lproj\AccountKitCountryCode.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\ur_PK.lproj\AccountKitCountryCode.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\vi.lproj\AccountKit.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\vi.lproj\AccountKit.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\vi.lproj\AccountKitCountryCode.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\vi.lproj\AccountKitCountryCode.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\zh.lproj\AccountKit.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\zh.lproj\AccountKit.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\zh.lproj\AccountKitCountryCode.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\zh.lproj\AccountKitCountryCode.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\zh_Hant_HK.lproj\AccountKit.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\zh_Hant_HK.lproj\AccountKit.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\zh_Hant_HK.lproj\AccountKitCountryCode.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\zh_Hant_HK.lproj\AccountKitCountryCode.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\zh_Hant_TW.lproj\AccountKit.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\zh_Hant_TW.lproj\AccountKit.strings</Link>
-    </BundleResource>
-    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\Resources\zh_Hant_TW.lproj\AccountKitCountryCode.strings">
-      <Link>Resources\AccountKitStrings.bundle\Resources\zh_Hant_TW.lproj\AccountKitCountryCode.strings</Link>
+    <BundleResource Include="..\..\externals\AccountKitStrings.bundle\**\*">
```

Comment:
Using wildcards and `%(RecursiveDir)` so we don't have to list EVERYTHING. Future-proof too.